src/module.ts

```diff
@@ -150,7 +150,7 @@ export default defineNuxtModule<ModuleOptions>({
     if (moduleOptions.exposeConfig) {
       const template = addTemplate({
         filename: 'tailwind.config.mjs',
-        getContents: () => `export default ${JSON.stringify(resolvedConfig, null, 2)}`
+        getContents: () => `${Object.entries(resolvedConfig).map(([k, v]) => `export const ${k} = ${JSON.stringify(v, null, 2)}`).join('\n')}\nexport default ${JSON.stringify(resolvedConfig, null, 2)}`
```

Comment:
Rather than include the config twice, can we reuse the named exports to produce the default export?

Reply:
A couple of ways I thought to do it - either I assign `Object.entries(resolvedConfig)` to a value or use `.keys` there instead, and if I should have used `.join` - as per you!